code/mob/living/critter/flock/flockbit.dm

```diff
@@ -20,7 +20,7 @@
 		src.zone_sel.change_hud_style('icons/mob/flock_ui.dmi')
 
 	src.name = "[pick_string("flockmind.txt", "flockbit_name_adj")] [pick_string("flockmind.txt", "flockbit_name_noun")]"
-	src.real_name = "[pick(consonants_upper)].[rand(10,99)].[rand(10,99)]"
+	src.real_name = src.flock ? src.flock.pick_name("flockbit") : name
```

Comment:
probably better to just make `flock_pick_name()` a global function than to have it as part of the flock datum, since it doesn't actually rely on any flock data.